dgraph/graphql/src/schema.js

```diff
@@ -1,6 +1,8 @@
 import { gql } from 'apollo-server-fastify';
 
 const typeDefs = gql`
+  scalar JSON
```

Comment:
😨.

Reply:
Haha, yeah I had an initial structure in mind and had mapped them in a spreadsheet, but it was sounding like we wanted all the data to be stored in mSupply and would then be synced out to all sites using mSupply's syncing). I think we may need a wider discussion on it.

This is what I originally had for the GraphQL Schema:
`  type DrugInteraction {
    name: String
    description: String
    severity: String }`
Possibly including the interacting drug's rxcui here as well

Reply:
> ```
> type DrugInteraction {
>     name: String
>     description: String
>     severity: String
>     source: String
>  }
> ```
> 
> Possibly including the interacting drug's rxcui here as well

Yeah, this is more like what I had in mind. I don't think we need to worry too much about what mSupply does with the data when it gets it (at least at this stage). Since mSupply will be getting the data via the legacy API, I think a good idea is to do they JSON conversion there (as part of this will need to update the legacy API to query GraphQL endpoints instead of Dgraph).